content/general/concepts/octal/octal.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: 'Octal' 
+Description: 'Octal is a number system base 8. It consists of figures from 0 to 7. Every figure in a number written in octal corresponds to 8 to the power of the certain digit.'
+Subjects: 
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Code Foundations'
+Tags: 
+  - 'Arithmetic'
+  - 'Encoding'
+CatalogContent: 
+  - 'paths/code-foundations'
+  - 'paths/computer-science'
+---
+
+The **octal** number system is base 8. The number system that we are used to is decimal, which is base 10. There are other number systems like [binary](https://www.codecademy.com/resources/docs/general/binary) or hexadecimal that work similarly. The octal system being base 8 means that there are only 8 possible digits: 0-7. Nevertheless, every number can be displayed.
+
+The octal system was often used in the past because systems were based on 12, 24, or 36 bits. The binary or hexadecimal system is commonly used in today's technologies, especially with 16, 32, or 64-bit machines. The octal number system has an advantage over the hexadecimal system in that we do not need to use characters from the alphabet to express a number in octal.
```

Comment:
```suggestion
Octal was often used because machines were based on 12, 24, or 36 bits. In contrast, binary or hexadecimal are commonly used in modern technologies, especially with 16, 32, or 64-bit machines. The octal number system has an advantage over the hexadecimal system in that characters from the alphabet are not used to express a number in octal.
```